### DISCUSSION TITLE
xcframework is too big

### DISCUSSION THREAD

**Discussion by sisoje** [onevcat/Kingfisher#1992] [Ideas]
Currently there are multiple platforms supported inside the xcframework (tvos watchos, macos...)

I would like to have iOS only framework zip archive... as it will be most used and much smaller that the one we currently have

**Comment by onevcat** [comment]
It is not the correct way to provide a framework like that. Furthermore, Xcode will pick up the correct slice for you when you add the xcframework to your project and build your app. So the final size in your app will only contain the necessary slice.

**Reply by sisoje** [reply]
I never complained about the size of the app, but the xcframework  download size.

For example `GoogleMaps` ships xcframeworks iOS only... Are you suggesting google are not doings things correctly?

We use some other xframeworks that also provide ios only version... its far from incorrect

**Comment by onevcat** [comment]
Starting with version 8.2.0, an iOS-specific xcframework variant is available on the release page.

Check the Assets section of the release page: https://github.com/onevcat/Kingfisher/releases/tag/8.2.0
